stable/phabricator/values.yaml

```diff
@@ -237,3 +237,15 @@ metrics:
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   ##
   # resources: {}
+
```

Comment:
Jus for comments convention, can you add a `##` line between the variable and the comment? And also use two # for comments:
```diff
+ ## Node labels for pod assignment
+ ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+ ##
  nodeSelector: {}
```

Reply:
Just updated.